packages/cojson/src/localNode.ts

```diff
@@ -178,12 +179,14 @@ export class LocalNode {
     crypto: CryptoProvider;
     initialAgentSecret?: AgentSecret;
     peers?: Peer[];
+    syncWhen?: SyncWhen;
```

Comment:
I'd prefer both params to be part of a single [SyncConfig](https://github.com/garden-co/jazz/blob/feat/resume-sync-on-restart/packages/jazz-tools/src/tools/types.ts#L56) object, but didn't want to introduce a breaking change in the local node API